examples/pytorch/dimenet/qm9.py

```diff
@@ -0,0 +1,218 @@
+"""QM9 dataset for graph property prediction (regression)."""
+import os
+import numpy as np
+import scipy.sparse as sp
+import dgl
+
+from tqdm import trange
+from dgl.data import DGLDataset
+from dgl.data.utils import download, _get_dgl_url, load_graphs, save_graphs
+from dgl.convert import graph as dgl_graph
+from dgl.transform import to_bidirected
+from dgl import backend as F
```

Comment:
Since this example is for PyTorch, you can directly use PyTorch APIs rather than `dgl.backend`.